docs/2101UserGuide.md

```diff
@@ -109,110 +109,98 @@ Examples:
 
 <hr class="feature-class-separator">
 
-### Event commands
+### Event Commands
+
+As a CCA head, keeping track of event details is a major responsibility. CCACommander allows you to do so with the event-related commands below:
 
 #### Create an Event : `createEvent`
 
-Creates a new event with accompanying details (name, location, date, tag).
+Creating a new event is no longer a hassle as it can be done with just one simple command - `createEvent`!
 
-Format: `createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]...`
+Here is how you can use it:<br>
+`createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]...`
 
-* Acceptable values for `DATE`: Dates in the format of `YYYY-MM-DD`.
+Here is an example command where we create an event **named** "Party" held at the **location** "Raffles Hall" on the
+**date** 2023-09-16, with **tag** "Dinner" and **tag** "Sem1":<br>
+`createEvent n/Party l/Raffles Hall d/2023-09-16 t/Dinner t/Sem1`
 
-Examples:
-* `createEvent n/Party l/Raffles Hall d/2023-09-16` creates an event `Party` in CCACommander.
+<figure>
+    <img src="images/CreateEventPostCommand.png"
+         alt="createEvent Post Command">
+    <figcaption>After executing the `createEvent` command</figcaption>
+</figure>
 
 <hr class="command-separator">
 
-#### Delete an Event: `deleteEvent`
+#### Edits an Event : `editEvent`
 
-Deletes the event at the specified index.
+If you forgot to note any important details, or realise that you made a mistake in any field of the event created,
+the `editEvent` command is here to help.
 
-Format: `deleteEvent EVENT_INDEX`
+Here is how you can use it:<br>
+`editEvent EVENT_INDEX [n/EVENT_NAME] [l/LOCATION] [d/DATE] [t/TAG]...`
 
-* Deletes the event at the specified `EVENT_INDEX`.
-* The index refers to the index number shown in the **currently displayed** event list.
-* The index **must be a positive integer** that is within the range of the length of the event list.
-
-Examples:
-* `deleteEvent 1` deletes the 1st event in the event list.
-* `deleteEvent 10` deletes the 10th event in the event list.
+Here is an example command where we edit an **event** at index 1, changing the **location** to be "MBS" and the **date** to be "2023-10-20".<br>
```

Comment:
I think index should be bolded instead of event, cause the parameteres are bolded so it's 
```suggestion
Here is an example command where we edit an event at **index** 1, changing the **location** to be "MBS" and the **date** to be "2023-10-20".<br>
```